cose-key-thumbprint/draft-ietf-cose-key-thumbprint.md

```diff
@@ -464,19 +464,19 @@ IANA is requested to add the following entry to the IANA "CWT Confirmation
 Methods" registry established by {{RFC8747}}:
 
 - Confirmation Method Name: ckt
-- Confirmation Method Description: COSE Key Thumbprint
+- Confirmation Method Description: COSE Key SHA-256 Thumbprint
 - JWT Confirmation Method Name: jkt

```

Comment:
```suggestion
- Confirmation Method Name: jkt

```

Reply:
Reading from the registration template in: https://datatracker.ietf.org/doc/html/rfc8747#section-7.2.1

```
JWT Confirmation Method Name:
Claim Name of the equivalent JWT confirmation method value, as registered in the "JSON Web Token Claims" subregistry in the "JSON Web Token (JWT)" registry [[IANA.JWT](https://www.iana.org/assignments/jwt)]. CWT claims should normally have a corresponding JWT claim. If a corresponding JWT claim would not make sense, the designated experts can choose to accept registrations for which the JWT Claim Name is listed as "N/A".
```

I think this is correct as is

Reply:
The intention is to keep JWT and CWT confirmation methods aligned, so CWT registrations are requested to cite JWT ones.